packages/reports/src/Containers/open-positions.jsx

```diff
@@ -505,31 +553,70 @@ const OpenPositions = ({
+                                    value={contract_type_value}
+                                    onChange={handleContractChange}
+                                />
+                            </div>
+
+                            {is_accumulator_selected && (
+                                <div className='open-positions__accumulator-container__rates-dropdown'>
+                                    <Dropdown
+                                        is_align_text_left
+                                        name='accumulator_rates'
+                                        list={accumulator_rates}
+                                        value={accumulator_rate}
+                                        onChange={handleRateChange}
+                                    />
+                                </div>
+                            )}
                         </div>
                     </DesktopWrapper>
                     <MobileWrapper>
-                        <SelectNative
-                            className='open-positions__contract-types-selector'
-                            list_items={contract_types.map(option => ({
-                                text: option.text,
-                                value: option.value,
-                            }))}
-                            value={contract_type_value}
-                            should_show_empty_option={false}
-                            onChange={handleChange}
-                        />
+                        <div
+                            className={
+                                is_accumulator_selected
+                                    ? 'open-positions__accumulator-container--mobile'
+                                    : 'open-positions__contract-types-selector-container--mobile'
+                            }
+                        >
+                            <SelectNative
+                                className='open-positions__accumulator-container-mobile__contract-dropdown'
+                                list_items={contract_types.map(option => ({
+                                    text: option.text,
+                                    value: option.value,
+                                }))}
+                                value={contract_type_value}
+                                should_show_empty_option={false}
+                                onChange={handleContractChange}
+                            />
+                            {is_accumulator_selected && (
+                                <SelectNative
+                                    className='open-positions__accumulator-container--mobile__rates-dropdown'
+                                    list_items={accumulator_rates.map(option => ({
+                                        text: option.text,
+                                        value: option.value.toString(),
+                                    }))}
```

Comment:
@henry-deriv please try this solution **here and also the same for the `list` at line 578**:
```suggestion
                                    list_items={accumulator_rates.map(option => ({
                                        text: option === 'all' ? localize('All Rates') : `${getGrowthRatePercentage(+option)}%`,
                                        value: option,
                                    }))}
```
for this to work, please replace `accumulator_rates` at line 363 with an array 🙂:
```
const accumulator_rates = ['all', '0.01', '0.02', '0.03', '0.04', '0.05'];
```